aspnet/fundamentals/distributed-cache.rst

```diff
@@ -0,0 +1,10 @@
+.. _distributed-cache:
+
+.. include:: /../common/stub-topic.txt
+
+|stub-icon| Working with a Distributed Cache
```

Comment:
@ardalis You need to include this new file in the TOC to fix the CI build error.